src/wp-includes/html/class-wp-html-attribute-token.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * HTML Tag Processor: Attribute token structure class.
+ *
+ * @package WordPress
+ * @subpackage HTML
```

Comment:
```suggestion
 * @subpackage HTML_API
```

This naming convention is used in the REST API and soon in Fonts API.

Why add `_API`?

The term "HTML" has a specific meaning of referring to the language itself or more broadly as the web page's markup. By itself, the subpackage does not clearly communicate that it is the API that processes HTML tags.

Adding `_API` suffix helps to provide clarity and context for contributors and extenders to better understand.